system-tests/projects/e2e/cypress/e2e/spec_bridge.cy.ts

```diff
@@ -0,0 +1,20 @@
+it('visits foobar.com and types foobar inside an input', () => {
```

Comment:
Not in particular. Main reason I didn't add it to the `driver` tests is the main test is really to test additional styles to the `app` package and behavior. I originally started with a `cy-in-cy` test inside the `app` package, but had issues configuring the host, plus it only ran in chrome and regressions would likely be a lot more obvious in electron. So I just moved it to a system test 🤷 .

Reply:
I'd prefer that this and the other ones be driver tests.

In general, it's better to use regular Cypress tests (driver tests, etc) if possible and reserve system tests for when it's not possible to write regular Cypress tests, like when the tests need specific project configuration, need dependencies the driver does not have, or when testing certain failures. It's much easier to iterate on driver tests as well as debug any failures, plus the results go to the Dashboard and utilize retries and config we already have set up for them. 

It's alright if the tests cover more than driver behavior. A lot of the driver tests are true e2e tests. Since this issue is specific to cy.origin(), I'd say the driver is as appropriate a place as any for the tests.